### ISSUE TITLE
LightGBMRanker Documentation

### ISSUE THREAD

**Issue by tranhan02** [microsoft/SynapseML#1831]
Is there any documentation on how to use LightGBMRanker? I have used objective:'lambdarank' and eval_metriv:'ndcg' but I don't know if the loss function is calculated by listwise or pairwise method in learning to rank problem.
Hope to get response as soon as possible ,because I am in a matter of greate urgency ! thanks very much.

**Comment by serena-ruan** [comment]
Hi @tranhan02 our LightGBMRanker wraps the official lightgbm ranker, so you could refer to their docs for any algorithm-based questions as well.(https://lightgbm.readthedocs.io/en/latest/pythonapi/lightgbm.LGBMRanker.html) But I found this stackoverflow page, hope it helps: https://stackoverflow.com/questions/68611032/is-it-possible-to-use-evaluation-metrics-like-ndcg-as-a-loss-function

**Comment by mhamilton723** [comment]
Attaching some links to our docs:

[LightGBM Ranking Example](https://microsoft.github.io/SynapseML/docs/features/lightgbm/LightGBM%20-%20Overview/#lightgbm-ranker)
[LightGBM on Spark overview](https://microsoft.github.io/SynapseML/docs/features/lightgbm/about/)
[LightGBM Ranking Quick Example and Refenernce](https://microsoft.github.io/SynapseML/docs/documentation/estimators/estimators_lightgbm/#lightgbmranker)

**Comment by mhamilton723** [comment]
Feel free to comment to reopen this issue if these docs are not sufficient
